src/test/java/model/line/LineTest.java

```diff
@@ -0,0 +1,49 @@
+package model.line;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import java.util.List;
+import java.util.stream.Stream;
+import model.bridge.Bridge;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class LineTest {
+    @DisplayName("라인 생성 시 다리가 겹치면 예외가 발생한다")
+    @ParameterizedTest
+    @MethodSource("provideDuplicatedBridges")
+    void testInvalidLine(List<Bridge> bridges) {
+        assertThatThrownBy(() -> new Line(bridges))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    private static Stream<Arguments> provideDuplicatedBridges() {
+        return Stream.of(
+                Arguments.of(createBridges(List.of(1, 1, 0))),
+                Arguments.of(createBridges(List.of(0, 1, 1, 0)))
+        );
+    }
+
+    @DisplayName("라인 생성 시 다리가 겹치지 않으면 예외가 발생하지 않는다")
```

Comment:
```suggestion
    @DisplayName("라인 생성 시 다리가 겹치지 않으면 성공")
```
예외라는 표현은 실제로 예외가 일어나는 테스트에만 사용해보는건 어떨까요?
테스트가 많아질수록 엄청 많은 테스트 제목을 한 번에 보게 되는데, 그럴 수록 제목이 확실하게 차별되는게 덜 헷갈리더라구요

Reply:
고려하지 못 했던 부분이었습니다. 바로 반영하겠습니다😊